namespace java thrift.test

struct myStringStruct {
  1: string a
}
